### ISSUE TITLE
`create-codemod-app/src/new-codemod-app-templates/.gitignore` does not exist?

### ISSUE THREAD

**Issue by hockeybuggy** [dangreenisrael/create-codemod-app#1]
Howdy. Long time user first time issue reporter.

```
➤ npx create-codemod-app                                                                                                                                                                                                                
? command name for this codemod app? dansux
Failed to generate codemod app
{ changes: [],
  failures:
   [ { type: 'add',
       path: '',
       error:
        'ENOENT: no such file or directory, stat \'/Users/danderson/.npm/_npx/18172/lib/node_modules/create-codemod-app/src/new-codemod-app-templates/.gitignore\'' },
     { type: 'add',
       path: '/Users/danderson/devel/{{packageName}}/package.json',
       error: 'Aborted due to previous action failure' },
     { type: 'add',
       path: '/Users/danderson/devel/{{packageName}}/bin/run-codemod.js',
       error: 'Aborted due to previous action failure' },
     { type: 'add',
       path:
        '/Users/danderson/devel/{{packageName}}/config/codemod-config.js',
       error: 'Aborted due to previous action failure' } ] }
```

Seems like something doesn't exist?

**Comment by dangreenisrael** [comment]
Fixed in 0.0.5 . Thanks @hockeybuggy

**Comment by hockeybuggy** [comment]
Nice! Thanks @dangreenisrael. Looks good.
```
➤ npx create-codemod-app
? command name for this codemod app? dansux

Setting up the app and running `npm install`

npm WARN deprecated cross-spawn-async@2.2.5: cross-spawn no longer requires a build toolchain, use it instead
npm WARN deprecated left-pad@1.3.0: use String.prototype.padStart()

> fsevents@1.2.9 install /Users/danderson/devel/dansux/node_modules/fsevents
> node install

node-pre-gyp WARN Using request for node-pre-gyp https download
[fsevents] Success: "/Users/danderson/devel/dansux/node_modules/fsevents/lib/binding/Release/node-v64-darwin-x64/fse.node" is installed via remote

> core-js@3.2.1 postinstall /Users/danderson/devel/dansux/node_modules/core-js
> node scripts/postinstall || echo "ignore"

Thank you for using core-js ( https://github.com/zloirock/core-js ) for polyfilling JavaScript standard library!

The project needs your help! Please consider supporting of core-js on Open Collective or Patreon:
> https://opencollective.com/core-js
> https://www.patreon.com/zloirock

Also, the author of core-js ( https://github.com/zloirock ) is looking for a good job -)

npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN dansux@1.0.0 No description
npm WARN dansux@1.0.0 No repository field.

added 882 packages from 467 contributors and audited 883290 packages in 15.409s
found 0 vulnerabilities

? Would you like to create a global 'dansux' command for running codemods? (this uses npm link) No


 Everything is setup in the 'dansux' directory
```
